src/translations/index.js

```diff
@@ -100,27 +100,27 @@ const TRANSLATIONS = {
         'export-finish': 'Finish Export',
         'export-more-options-heading': 'More export options',
 
-        'remove-key-log-out': 'Log out',
-        'remove-key-intro-text': 'Logging out means removing your Wallet File from this browser. '
-                               + 'Make sure you have it stored somewhere, or at least have your '
-                               + 'Recovery Words accessible.',
-        'remove-key-intro-text-red': 'If you have neither of them, there’s no chance to regain '
-                                   + 'access to your wallet.',
-        'remove-key-key-file': 'Wallet File',
-        'remove-key-download-key-file': 'Download Wallet File',
-        'remove-key-show-recovery-words': 'Show Recovery Words',
-        'remove-key-confirm': 'Log out of your wallet',
+        'remove-key-heading': 'Don\'t lose access',
+        'remove-key-intro-text': 'If you log out without saving your account, you will irretrievably '
+                                + 'lose access to it!',
+        'remove-key-login-file': 'LoginFile',
+        'remove-key-login-file-question': 'Is your LoginFile savely stored and accessible?',
+        'remove-key-download-login-file': 'Download LoginFile',
+        'remove-key-recovery-words-question': 'Do you know where your Recovery Words are?',
+        'remove-key-show-recovery-words': 'Create a backup',
+        'remove-key-first-confirm': 'I can log in again',
```

Comment:
Missing in `de:`.
Also not the same as in DOM: `I am able to log in again`. 

I wonder why checklangs does not mention it.